RecoEgamma/EgammaPhotonProducers/src/ReducedEGProducer.cc

```diff
@@ -568,6 +614,39 @@ void ReducedEGProducer::produce(edm::Event& theEvent, const edm::EventSetup& the
 
   for (reco::GsfElectron &gsfElectron : *gsfElectrons) {
     relinkGsfElectronCore(gsfElectron, gsfElectronCoreMap, outgsfElectronCoreHandle);
+
+    // -----
+    // Also in this loop let's reling ambiguous tracks
+    // -----
+    std::vector<reco::GsfTrackRef> ambigTracksInThisElectron;
+    // Here we loop over the ambiguous tracks and save them in a vector
+    for (reco::GsfTrackRefVector::const_iterator igsf = gsfElectron.ambiguousGsfTracksBegin(); igsf != gsfElectron.ambiguousGsfTracksEnd(); ++igsf) {
```

Comment:
I do not know how to do that for these objects (ambiguous trakcs).
If you know how - please do that. Otherwise I think it's fine as it is.